### PR TITLE
Add `cads_collection_radio_buttons` method

### PR DESCRIPTION
**Pull request by leeky**

### CODE REVIEW

File: lib/citizens_advice_form_builder/elements/collections/radio_buttons.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  module Elements
+    module Collections
+      class RadioButtons < Base
+        include ActionView::Helpers::FormOptionsHelper
+
+        def render
+          component = CitizensAdviceComponents::RadioGroup.new(
+            legend: label,
+            name: field_id,
+            options: {
```
Comment (marianayovcheva) [inline lib/citizens_advice_form_builder/elements/collections/radio_buttons.rb:13]: Would it be possible to add `layout` option here? There are a few places in the designs where the radio buttons are inlined. If it's difficult to add as part of this PR, it's non blocking as I think we can live without inline styles for those for now.

File: lib/citizens_advice_form_builder/elements/collections/radio_buttons.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+module CitizensAdviceFormBuilder
+  module Elements
+    module Collections
+      class RadioButtons < Base
+        include ActionView::Helpers::FormOptionsHelper
+
+        def render
+          component = CitizensAdviceComponents::RadioGroup.new(
+            legend: label,
+            name: field_id,
```
Comment (marianayovcheva) [inline lib/citizens_advice_form_builder/elements/collections/radio_buttons.rb:12]: A big aside, I think the recent design system v5.7.0 will impact how the name/id works once it's rolled out

**Review by marianayovcheva — APPROVED** [review]
📻 LGTM!

**Review by WagnerMatos — APPROVED** [review]

File: lib/citizens_advice_form_builder/elements/collections/radio_buttons.rb (hunk 2 above)
Comment (leeky) [reply lib/citizens_advice_form_builder/elements/collections/radio_buttons.rb:12]: Good point!

File: lib/citizens_advice_form_builder/elements/collections/radio_buttons.rb (hunk 1 above)
Comment (leeky) [reply lib/citizens_advice_form_builder/elements/collections/radio_buttons.rb:13]: It's a small enough change, but I think I'll merge this as-is and add a ticket to add this separately.

**Pull request merged**